src/lib/frontend/cnf.mli

```diff
@@ -31,10 +31,10 @@
 (* used in the typechecker for the text-mode *)
 val make :
   Commands.sat_tdecl list ->
-  (int Typed.tdecl, 'a) Typed.annoted ->
+  ('a Typed.tdecl, 'a) Typed.annoted ->
   Commands.sat_tdecl list
 
 (* used in the GUI *)
 val make_list :
-  (int Typed.tdecl, 'a) Typed.annoted list ->
+  ('a Typed.tdecl, 'a) Typed.annoted list ->
```

Comment:
Should this be `('a Typed.tdecl, 'b) Typed.annoted`?

Reply:
No it must be the same type. I changed it by the aliad defined in typed (`'a Typed.atdecl = 'a Typed.tdecl, 'a`)

Reply:
Hmm OK. I don't understand why `(int Typed.tdecl, 'a) Typed.annotated` worked before then :)  Anyways, using `atdecl` sounds good yeah, let's merge.